packages/website/src/content/docs/main/sideNavSections.ts

```diff
@@ -491,6 +491,11 @@ export const SIDENAV_SECTIONS: TopLevelDocsSectionMetadata[] = [
                 path: "/kubernetes",
                 sub: makeModuleDir(modules.modules, "kubernetes", "direct"),
               },
+              {
```

Comment:
Let's put this in the authentik category since we are not going to be developing any more mongodb modules.